app/code/Vaimo/QtyManager/view/frontend/web/js/update-row-price.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright © Vaimo, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+define([
+   'jquery',
+   'Magento_Checkout/js/model/quote',
+   'Magento_Catalog/js/price-utils',
+   'jquery/ui',
+], function ($, quote, priceUtils) {
+    'use strict';
+
+    $.widget('mage.updateRowPrice', {
+        options: {
+            debounceDelay: 1000,
+        },
+
+        _create: function () {
+            this._constructorBlock();
+            this.extendQtyManager();
+        },
+
+        /**
+         * @description Method for adding increase/decrease buttons template and variable declaration.
+         * And replace quantity input to this template.
+         */
+        _constructorBlock: function () {
+            this.currentProductRow = this.element.closest('.item-info');
+            this.currentProductPrice = this.currentProductRow.find('[data-th="Price"] .price');
+            this.currentProductTotal = this.currentProductRow.find('[data-th="Subtotal"] .price');
```

Comment:
This will be fucked up if u set different translation. Better find untranslatable selector.